cmake/section_relocation.cmake

```diff
@@ -0,0 +1,11 @@
+function(zephyr_kernel_code_relocate file location)
+  get_property(CODE_DATA_RELOCATION_STRINGS_LOCAL GLOBAL PROPERTY
+   CODE_DATA_RELOCATION_STRINGS)
+  if(NOT ${CODE_DATA_RELOCATION_STRINGS_LOCAL} STREQUAL '')
```

Comment:
It is much simpler if you maintain this list as a list instead of a string, by using APPEND instread of APPEND_STRING.

And then do a string replace just before you need to convert to a comma separated string.

Reply:
done, pls check